scripts/updateSources.js

```diff
@@ -159,33 +123,12 @@ async function main() {
       return content;
     }
 
-    // Dialog: Workaround for https://github.com/vaadin/vaadin-confirm-dialog-flow/issues/136
-    // Since this project contains a dependency to vaadin-confirm-dialog, the height is different
-    // and the tests fail.
-    content = ignore_test_method(content, source, 'DialogIT.openAndCloseBasicDialog_labelRendered');
-    content = ignore_test_method(content, source, 'ServerSideEventsIT.chartClick_occured_eventIsFired');
-    content = ignore_test_method(content, source, 'GridDetailsRowIT.gridUpdateItemUpdateDetails');
-    content = ignore_test_method(content, source, 'BasicIT.customComboBox_circularReferencesInData_isEdited');
-    content = ignore_test_method(content, source, 'BasicIT.customComboBoxIsUsedForEditColumn');
-    content = ignore_test_method(content, source, 'BasicIT.checkboxEditorIsUsedForCheckboxColumn');
-
     content = ignore_test_method(content, source, 'EditOnClickIT.editButtonsAreHiddenIfEditOnClickIsEnabled');
-    content = ignore_test_method(content, source, 'RendererIT.testRenderer_initialComponentRendererSet_rendersComponentsThatWork');
-    content = ignore_test_method(content, source, 'RendererIT.testRenderer_componentRendererSet_rendersComponentsThatWork');
```

Comment:
These tests were also re-enabled in https://github.com/vaadin/flow-components/pull/280.